Add a note about custom logs vs devtools logs

KevinU3 opened the pull request:
Added more notes to inform developers that uses an online IDE about the location of the timing logs.

<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding details below will help us to merge your PR faster. -->

### Description

Some online IDE does not log the timing methods on their own console. Suggested opening DevTools.

### Motivation

Inform developers who uses an online IDE on where to look for the timing logs.

### Additional details

<!-- 🔗 Link to release notes, vendor docs, bug trackers, source control, or other places providing more context -->

### Related issues and pull requests

<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->


<!-- 👷‍♀️ After submitting, go to the "Checks" tab of your PR for the build status -->

KevinU3 committed:
Notes to the timer methods

Added more notes to inform developers that uses an online IDE about the location of the timing logs.

Elchi3 commented:
@KevinU3 Thanks for your PR! 

"the timing methods do not log to the console of some online IDE."

I think "some online IDEs" can always decide how they implement the console standard. I'm not sure how useful the note is. And while this is true for the IDE you have in mind and for the timing methods there, I guess it could be generalized to say that certain websites/online IDEs re-implement `console` objects and that they don't necessarily come with all of the features that are in the browser's DevTools implementation of `console`. Does that make sense to you?

KevinU3 commented:
> @Elchi3 Thanks for getting back to me!
> 
> "don't necessarily come with all of the features that are in the browser's DevTools implementation of `console`"

While I agree with this, I think it is still best to be explicit about stating the reason for the logs not displaying (this could be the reason you stated) and how to resolve it (find the logs). This is especially important in the "guide" section of the console API doc. Don't you agree?

Elchi3 commented:
Yes, I think it's a good idea to note this! Should we then make the note box a bit more generic and place it at the top rather than in a section specific to timers? I would expect a wording like this to explain the situation (feel free to improve):

> Note: Certain online IDEs and editors re-implement the `console` API and behave similar to the browser's `console` object. However, not all functionality might be provided there (for example, you might not get correct timing logs). Open the browser's DevTools to access the browser's `console` API as documented here.

KevinU3 committed:
Update index.md

I agree, the note should be specified at the top. 

I wouldn't be so sure that some online IDEs and editors "re-implement" the console API, so I think using the word "may" would be more appropriate.

Elchi3 commented on files/en-us/web/api/console/index.md:
```diff
@@ -26,6 +26,8 @@ gives a few [Usage](#usage) examples.
 
 {{AvailableInWorkers}}
 
+> **Note:** Certain online IDEs and editors may implement the console API differently than the browsers. As a result, certian functionality of the console API such as the timer methods, may not be outputted in the console of some online IDE or editors. Open your browser's DevTools console to see the logs as shown in this documentation.
```
```suggestion
> **Note:** Certain online IDEs and editors may implement the console API differently than the browsers. As a result, certain functionality of the console API, such as the timer methods, may not be outputted in the console of online IDEs or editors. Always open your browser's DevTools console to see the logs as shown in this documentation.
```

Elchi3 committed:
typo fixes

Elchi3 commented on files/en-us/web/api/console/index.md:
```diff
@@ -242,6 +244,7 @@ Notice that the timer's name is displayed both when the timer is started and whe
 > **Note:** It's important to note that if you're using this to log the timing for network traffic, the timer will report the total time for the transaction, while the time listed in the network panel is just the amount of time required for the header.
 > If you have response body logging enabled, the time listed for the response header and body combined should match what you see in the console output.
 
+
```
```suggestion
```

Elchi3 committed:
rm blank line

Elchi3 committed:
Merge branch 'main' into patch-1

Elchi3 reviewed (approved):
Thank you! 👍
